New Backoffice: Preview004 (#5804)
umbraco/UmbracoDocs: new-backoffice/tutorials/creating-your-first-extension.md

This guide will help you set up your first extension using vanilla JavaScript or Vite, Typescript, and Lit. It is also part of the prerequisites for [Creating a Property Editor](creating-a-property-editor/) and [Creating a Custom Dashboard](creating-a-custom-dashboard.md) tutorials.

> You should read the [Setup Your Development Environment](../extending/development-flow/README.md) article before you start this tutorial.

Reviewer: ```suggestion
Before following this tutorial make sure to read the [Setup Your Development Environment](../extending/development-flow/README.md) article.
```
